packages/indexer-common/src/network.ts

```diff
@@ -127,31 +127,40 @@ export class Network {
       operator: wallet.address,
     })
 
-    const paused = await monitorNetworkPauses(logger, contracts, networkSubgraph)
-    const isOperator = await monitorIsOperator(logger, contracts, indexerAddress, wallet)
+    try {
+      const paused = await monitorNetworkPauses(logger, contracts, networkSubgraph)
+      const isOperator = await monitorIsOperator(
+        logger,
+        contracts,
+        indexerAddress,
+        wallet,
+      )
 
-    return new Network(
-      logger,
-      wallet,
-      indexerAddress,
-      indexerUrl,
-      geoCoordinates,
-      contracts,
-      networkSubgraph,
-      ethereum,
-      paused,
-      isOperator,
-      restakeRewards,
-      rebateClaimThreshold,
-      rebateClaimBatchThreshold,
-      rebateClaimMaxBatchSize,
-      poiDisputeMonitoring,
-      poiDisputableEpochs,
-      gasIncreaseTimeout,
-      gasIncreaseFactor,
-      baseFeePerGasMax,
-      maxTransactionAttempts,
-    )
+      return new Network(
+        logger,
+        wallet,
+        indexerAddress,
+        indexerUrl,
+        geoCoordinates,
+        contracts,
+        networkSubgraph,
+        ethereum,
+        paused,
+        isOperator,
+        restakeRewards,
+        rebateClaimThreshold,
+        rebateClaimBatchThreshold,
+        rebateClaimMaxBatchSize,
+        poiDisputeMonitoring,
+        poiDisputableEpochs,
+        gasIncreaseTimeout,
+        gasIncreaseFactor,
+        baseFeePerGasMax,
+        maxTransactionAttempts,
+      )
+    } catch (error) {
+      throw indexerError(IndexerErrorCode.IE004, error.message)
```

Comment:
Is this try..catch needed since it will still throw an error? Shouldn't the functions here: monitorNetworkPause() and monitorIsOperator() throw useful, specific errors on their own?

Reply:
this catches the same case on the other comment on `suberror: IndexerErrorCode.IE007` for `monitorNetworkPause()`

Without both catches, we get an error like 
```
Error: call revert exception [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (method="paused()", data="0x", errorArgs=null, errorName=null, errorSignature=null, reason=null, code=CALL_EXCEPTION, version=abi/5.6.1)
    at Logger.makeError 
``` . 

With the other handle, IE007 is probably enough. I wanted to point people towards IE004 errors markdown because I thought it was helpful but not referenced elsewhere, so the error message would have 
```
[1662231687713] ERROR (IndexerAgent/4838 on book): Failed to check for network pause with contracts controller
    ...
   suberror: "IE007"
    cause: "call revert exception ..."
IndexerError: Failed to synchronize with network
     code: 'IE004',
  explanation: 'https://github.com/graphprotocol/indexer/blob/master/docs/errors.md#ie004',
  cause: 'Failed to check for network pause'
}
```

(I will remove the catch for `IE004` though, I can see how it is redundant